terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -237,13 +237,13 @@ resource "google_compute_instance_template" "application" {
           name  = local.application_name != null ? local.application_name : var.image
           image = "${var.container_registry}/${var.image_repo}/${var.image}:${var.image_tag}"
           env   = local.environment_variables
+          securityContext = {
+            privileged = true # For loading eBPF programs
```

Comment:
```suggestion
            privileged : true # For loading eBPF programs
```

Reply:
At least from the example given, this seems to be the correct syntax.

Reply:
The example doesn't use `yamlEncode` though?

Reply:
And we don't use `:` anywhere else for assigning values.

Reply:
Seems like it actually doesn't matter. Just a style difference.